lib/Language/Haskell/Stylish/Step/Imports.hs

```diff
@@ -103,6 +119,27 @@ data LongListAlign
     | Multiline -- multiline
     deriving (Eq, Show)
 
+data Pattern = Pattern
+  { regex  :: Regex
```

Comment:
it's fairly clear from reading the code, but perhaps worth a comment that this is a cached computed field derived from `string`.